### ISSUE TITLE
`module_name` fails if `.py` filename begins with `mkdocs_`

### ISSUE THREAD

**Issue by mcrumiller** [fralau/mkdocs-macros-plugin#254]
I was trying to use something other than `main.py` with the following configuration:

```yaml
plugins:
  ...
  - macros:
        module_name: mkdocs_macros
```
with an accompanying `mkdocs_macros.py` file. Upon trying to `mkdocs serve` the site, I ran into this error:

```
ModuleNotFoundError: No module named 'mkdocs_macros.plugin'; 'mkdocs_macros' is not a package
```

If I rename the file to `mkdocsmacros.py` (no underscore) and update the `mkdocs.yml` file, it works properly. It also works if I use something like `my_macros.py`. It appears that modules that begin with `mkdocs_` cause the issue.

I think a simple update to the User Guilde/documentation warning to avoid this pattern may help.

**Comment by fralau** [comment]
Thanks for the heads ups. Have you tried other names with an underscore?

**Comment by mcrumiller** [comment]
> Thanks for the heads ups. Have you tried other names with an underscore?

You're right, shame on me, that's obvious...it looks like `mkdocs_` is off limits, not necessarily the underscore. Using an underscore is fine. I'll update the OP.
